### PR TITLE
Added admin column to users, made it possible for admin to delete any…

### PR DESCRIPTION
**Pull request by shanekrusen**
… post

### CODE REVIEW

**Comment by shanekrusen** [comment]
I can't tell what is failing the build, I don't have access.

**Comment by kenmazaika** [comment]
Looks like the test suite is failing.  Also would be nice to add an additional test, to ensure that admins can destroy items.

```
/home/rof/.rvm/rubies/ruby-2.2.3/bin/ruby -I/home/rof/cache/bundler/ruby/2.2.0/gems/rspec-core-3.4.4/lib:/home/rof/cache/bundler/ruby/2.2.0/gems/rspec-support-3.4.1/lib /home/rof/cache/bundler/ruby/2.2.0/gems/rspec-core-3.4.4/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb
...............................F......

Failures:

1) SharesController shares#destroy should allow a user to destroy shares
Failure/Error: expect(response).to redirect_to shares_path
  Expected response to be a <redirect>, but was <403>
# ./spec/controllers/shares_controller_spec.rb:151:in `block (3 levels) in <top (required)>'

Finished in 0.71093 seconds (files took 2.14 seconds to load)
38 examples, 1 failure

Failed examples:

rspec ./spec/controllers/shares_controller_spec.rb:147 # SharesController shares#destroy should allow a user to destroy shares
```

**Comment by shanekrusen** [comment]
This is ready for review/merge.

File: spec/controllers/shares_controller_spec.rb [hunk 1]
```diff
@@ -150,6 +150,16 @@
     delete :destroy, id: share.id
     expect(response).to redirect_to shares_path
    end
+   
+   it "should allow an admin to destroy shares" do
+    admin = FactoryGirl.create(:user)
+    admin.admin = true
+    admin.save!
```
Comment (kenmazaika) [inline spec/controllers/shares_controller_spec.rb]: As a heads up:

```
admin = FactoryGirl.create(:user, admin: true)
```

Does the same as the previous 3 lines in a single one.  Can be useful to shorten things.

**Comment by kenmazaika** [comment]
Nice work, Shane.  Looks great.

**Pull request merged**